src/components/AccountSeed.js

```diff
@@ -17,154 +17,93 @@
 'use strict';
 
 import React, { Component } from 'react';
-import { Animated, StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View } from 'react-native';
 import colors from '../colors';
-import { WORDS, WORDS_INDEX } from '../util/account';
+import PARITY_WORDS from '../../res/parity_wordlist.json';
+import BIP39_WORDS from '../../res/bip39_wordlist.json';;
 import TextInput from './TextInput';
 import TouchableItem from './TouchableItem';
 
 export default class AccountSeed extends Component {
   state = {
-    cursorPosition: 0,
-    keyboard: false,
-    animation: false,
-    suggestionsHeight: new Animated.Value(0) // Initial value for opacity: 0
+    useBIP39WordList: true,
+    useParityWordList: true,
   };
 
-  constructor(...args) {
-    super(...args);
-    this.getSuggestions = this.getSuggestions.bind(this);
-    this.handleCursorPosition = this.handleCursorPosition.bind(this);
-    this.getWordPosition = this.getWordPosition.bind(this);
-    this.keyboardDidShow = this.keyboardDidShow.bind(this);
-    this.keyboardDidHide = this.keyboardDidHide.bind(this);
-  }
-
-  keyboardDidShow() {
-    this.setState({ keyboard: true, animation: true });
-    Animated.timing(
-      // Animate over time
-      this.state.suggestionsHeight, // The animated value to drive
-      {
-        toValue: 35, // Animate to opacity: 1 (opaque)
-        duration: 200 // Make it take a while
-      }
-    ).start(() => this.setState({ animation: false }));
-  }
-
-  keyboardDidHide() {
-    this.setState({ animation: true });
-    Animated.timing(
-      // Animate over time
-      this.state.suggestionsHeight, // The animated value to drive
-      {
-        toValue: 0, // Animate to opacity: 1 (opaque)
-        duration: 200 // Make it take a while
-      }
-    ).start(() => {
-      this.setState({ keyboard: false, animation: false });
-    });
-  }
-
-  handleCursorPosition({
-    nativeEvent: {
-      selection: { start, end }
-    }
-  }) {
-    if (start !== end) {
-      return;
-    }
-    this.setState({ cursorPosition: start });
-  }
-
-  getSearchInput() {
-    const { value } = this.props;
-    const { cursorPosition } = this.state;
-    let startFrom = cursorPosition;
-    // find the first space or start of string
-    while (startFrom > 0 && [' '].indexOf(value.charAt(startFrom - 1)) === -1) {
-      --startFrom;
-    }
-    return value.substring(startFrom, cursorPosition);
-  }
-
-  getWordPosition() {
```

Comment:
What if user wants to make a correction in the middle? Why remove?

Reply:
I really thought twice before removing it. It's a matter of making things simple. There was a ton of complex logic just for this use case, and I didn't think it was worth it.

Reply:
I've reintroduced this feature in last commit here, logic is streamlined so it should make everyone happy :).